packages/office-addin-mock/src/host.ts

```diff
@@ -0,0 +1,23 @@
+import { ObjectData } from "./objectData";
+
+export enum Host {
+  excel = "excel",
+  outlook = "outlook",
+  powerpoint = "powerpoint",
+  word = "word",
+  other = "other",
+  notFound = "notFound",
+}
```

Comment:
I don't think we should have the "other" option.  It should something we know or unknow (which should be used instead of "notFound).  We should also include the other apps that Office.js supports . . . we may focus on these 4 a lot, but I believe there are other, less used, hosts that can also work.

Reply:
Sure! I just added all the supported hosts (following documentation) and changed the other/noFound to unknow!

Reply:
Looks like I spelled that wrong (left off a letter) and you used that bad spelling :-).  Is should be "unknown"

Reply:
You don't really need an enum option for other or notFound. Just use `undefined` for that case rather than the enum value, e.g. the type would be "Host | undefined". By the way, there is a similar enum in office-addin-debugging or dev-settings.

Reply:
Sure, I am now using the existing enum for hosts, which was in "manifest" package.